File: server/src/main/java/com/application/lamion/repository/RequestRepository.java
package com.application.lamion.repository;

import com.application.lamion.model.Request;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RequestRepository extends JpaRepository<Request, Long> {
    List<Request> findAllByEventIdOrderByIdDesc(long eventId);
}

File: server/src/main/java/com/application/lamion/repository/RequestRepository.kt
package com.application.lamion.repository

import com.application.lamion.model.Request
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RequestRepository : JpaRepository<Request?, Long?> {
    fun findAllByEventIdOrderByIdDesc(eventId: Long): List<Request?>?
}
